action/protocol/execution/evm/evm.go

```diff
@@ -279,6 +280,10 @@ func ExecuteContract(
 			sharedGas                 uint64
 			sharedGasFee, totalGasFee *big.Int
 		)
+		// if gas price is zero, set it to smallest unit
```

Comment:
no hard fork?

Reply:
deleted  it will fix gasprice in another pr.